tests/unit/hooks/command.py

```diff
@@ -0,0 +1,49 @@
+"""
+    Command interface for os.execve hook from EntrypointHook.
+
+    Store arguments and environ for hooked commands, used to create
+    both the command of entrypoint.py called during tests,
+    and the expected command for test comparison.
+"""
+
+import difflib
+import json
+
+class CommandNotFound(Exception):
+    """Raised when entrypoint command is not found or test fail."""
+
+class Command:
+    """
+    Represent a single execve command, with
+    its arguments and environment.
+
+    Can represent an entrypoint hooked command, the expected
+    result or the input of a test.
+    """
+    def __init__(self, argv, environ):
+        # Sort cli arguments to facilitate comparaison
+        sorted_args = argv[1:]
+        sorted_args.sort()
```

Comment:
Although this makes it easier to compare the list, the position of an argument passed to dogecoind matters, so this will give misleading results. Can we remove the sort?